drivers/power/adp1050/README.rst

```diff
@@ -245,8 +248,9 @@ Debug Attributes
 * ``status_temperature - TEMPERATURE status byte value of the device``
 * ``status_cml - CML status byte value of the device``
 * ``status_word - Status word value of the device``
+* ``status_iout - IOUT status byte value of the device``
 
-ADP1050 IIO Driver Initialization Example
+ADP105x IIO Driver Initialization Example
```

Comment:
changes from `ADP1050` to `ADP105x` (this and some above) are unnecessary

Reply:
copy